### PR TITLE
fix(text): tight option

### PR DESCRIPTION
**Pull request by rachelbt**
- added a tight option for titles with no margin needed
- fixed the login example

### CODE REVIEW

File: components/text/src/vwc-text-base.ts [hunk 1]
```diff
@@ -24,10 +24,14 @@ export class VWCTextBase extends LitElement {
 	@property({ type: String, reflect: true })
 		connotation?: TextConnotation;
 
+	@property({ type: Boolean, reflect: true })
+	tight = false;
```
Comment (yinonov) [inline components/text/src/vwc-text-base.ts:28]:
```suggestion
		tight = false;
```

File: components/text/readme.md [hunk 2]
```diff
@@ -52,8 +52,12 @@ On the other hand, 'vwc-text' can nest within semantic tags instead -
 | Property   | Attribute   | Type                                                                                                                                                                                                                                                                                                                                      |
 | ---------- | ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `fontFace` | `font-face` | `body-1` \| `body-1-bold` \| `body-1-code` \| `body-1-link` \| `body-2` \| `body-2-bold` \| `body-2-code` \| `body-2-link` \| `button` \| `button-dense` \| `button-enlarge` \| `caption` \| `caption-bold` \| `caption-code` \| `caption-link` \| `headline-1` \| `headline-2` \| `subtitle-1` \| `subtitle-2` \| `title-1` \| `title-2` |
+| `noSpacing` | `noSpacing` | `boolian` |
```
Comment (yinonov) [inline components/text/readme.md:55]:
```suggestion
| `noSpacing` | `noSpacing` | `boolean` |
```

File: components/text/src/vwc-text-base.ts [hunk 3]
```diff
@@ -24,10 +24,14 @@ export class VWCTextBase extends LitElement {
...
 		connotation?: TextConnotation;
 
+	@property({ type: Boolean, reflect: true })
+		tight = false;
+
 	protected getRenderClasses(): ClassInfo {
 		return {
 			[`connotation-${this.connotation}`]: !!this.connotation,
 			[`font-face-${this.fontFace}`]: !!this.fontFace,
+			['tight']: !!this.tight,
```
Comment (yinonov) [inline components/text/src/vwc-text-base.ts:34]:
```suggestion
			tight: this.tight,
```

**Review by yinonov — CHANGES_REQUESTED** [review]
@rachelbt the text snapshot is updated here so once merged, it will remove the buttons' PR difference of that snapshot

File: components/text/readme.md [hunk 4]
```diff
@@ -52,8 +52,12 @@ On the other hand, 'vwc-text' can nest within semantic tags instead -
 | Property   | Attribute   | Type                                                                                                                                                                                                                                                                                                                                      |
 | ---------- | ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `fontFace` | `font-face` | `body-1` \| `body-1-bold` \| `body-1-code` \| `body-1-link` \| `body-2` \| `body-2-bold` \| `body-2-code` \| `body-2-link` \| `button` \| `button-dense` \| `button-enlarge` \| `caption` \| `caption-bold` \| `caption-code` \| `caption-link` \| `headline-1` \| `headline-2` \| `subtitle-1` \| `subtitle-2` \| `title-1` \| `title-2` |
+| `noSpacing` | `noSpacing` | `boolean` |
```
Comment (yinonov) [inline components/text/readme.md:55]: @rachelbt how does flat sound? really struggling here 😝

**Comment by rachelbt** [comment]
don't think fat is right. I think we should either use tight yo noSpacing

File: components/text/readme.md (hunk 4 above)
Comment (yinonov) [reply components/text/readme.md:55]:
```suggestion
| `tight` | `tight` | `boolean` |
```

**Review by yinonov — APPROVED** [review]

**Pull request merged**